app/models/user_session.rb

```diff
@@ -7,6 +7,7 @@ class UserSession < ApplicationRecord
   has_one :account, through: :user
 
   validates :user_id, :key, presence: true
+  validates :ip, length: {maximum: 255}
```

Comment:
IP that long? Isn't there a type for ips?

Reply:
In the schema it is that long :
https://github.com/3scale/porta/blob/4b1d9bf7d223a27e74afc81fcc3a1612d1447f6c/db/schema.rb#L1392

Reply:
And we have another ip that we do the same:
https://github.com/3scale/porta/blob/4b1d9bf7d223a27e74afc81fcc3a1612d1447f6c/db/schema.rb#L1435